### PR TITLE
Update to SSK 2.0

### PR DESCRIPTION
**Pull request by mkelan**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,10 +1,35 @@
...
-# MSDyn365Commerce_BASEURL=<Retail Server Base URL Here>
-# MSDyn365Commerce_CHANNELID=<Retail Server Channel ID Here>
-# MSDyn365Commerce_CATALOGID=<Retail Server Catalog ID Here>
-# MSDyn365Commerce_OUN=<Retail Server OUN Here>
-# MSDyn365Commerce_BASEIMAGEURL=<Retail Server Base Image URL Here>
+# MSDyn365_HOST variable defines the domain name of your customer-facing e-Commerce site. 
+#   When this variable is set, if the URL https://localhost:4000/ is opened in a development environment, 
+#   your e-Commerce site will be rendered locally. If your site is protected through Azure Active Directory 
+#   (Azure AD) credentials, a prompt for a user name and password will appear.
+MSDyn365_HOST=
```
Comment (samjarawan) [inline .env]: Did you verify that an empty variable like this will be treated as if it's not set at all?

File: .env [hunk 2]
```diff
@@ -1,10 +1,35 @@
...
+# MSDyn365Commerce_CATALOGID variable defines the catalog ID for the online store that you're connecting to. 
+#   Currently, only the value 0 (zero) is supported.
+# MSDyn365Commerce_CHANNELID variable defines the online channel that you're connecting to.
+MSDyn365Commerce_BASEURL=
+MSDyn365Commerce_CHANNELID=
+MSDyn365Commerce_CATALOGID=
+MSDyn365Commerce_OUN=
+
+# The MSDyn365Commerce_BASEIMAGEURL variable defines the URL for a website's image assets. 
+# The URL follows a pattern and must be manually generated. Follow one of these steps:
```
Comment (samjarawan) [inline .env]: I think this is a cut/paste issue, perhaps mention.  "For more information see online product documentation"

File: package.json [hunk 3]
```diff
@@ -39,12 +39,10 @@
     "typeson": "5.10.1"
   },
   "dependencies": {
-    "@msdyn365-commerce-modules/data-types": "1.62.9",
-    "@msdyn365-commerce-modules/shared-components": "1.62.9",
+    "@msdyn365-commerce-modules/starter-pack": "2.2.0-rc.20708541.0",
```
Comment (samjarawan) [inline package.json]: why does the version number look so different with the "rc..." string in it?

File: src/modules/helloworld/helloworld.props.autogenerated.ts [hunk 4]
```diff
@@ -6,7 +6,7 @@
 
 import * as Msdyn365 from '@msdyn365-commerce/core';
 
-export interface IHelloworldConfig {
+export interface IHelloworldConfig extends Msdyn365.IModuleConfig {
     showText?: string;
```
Comment (samjarawan) [inline src/modules/helloworld/helloworld.props.autogenerated.ts]: This is an aside, but can we change the variable "showText" to something more realistic like "title", in fact i meant to ask for this and pre-emtively changed docs to show title :-)

**Comment by siddthesciencekid** [comment]
Looks good to me!

**Comment by mkelan** [comment]
PR comments addressed. Also updated the SOFTWARE LICENSE TERMS as well

**Pull request merged**